angular/projects/admin-nrpti/src/app/services/record.service.ts

```diff
@@ -46,12 +46,24 @@ export class RecordService {
     return this.http.post<object>(`${this.apiService.pathAPI}/${queryString}`, record, {});
   }
 
-  createRecord(record: object): Observable<object> {
+  createRecord(record: any): Observable<object> {
+    // We handle document logic when we add or remove documents not when we add/edit record
+    for (const property of Object.keys(record)) {
+      record[property].forEach(element => {
+        delete element.documents;
+      });
+    }
```

Comment:
We shouldn't have to iterate through all the documents on the client side if the API just ignores it.

Reply:
Fixed in c21738f